packages/react-storage/src/components/StorageBrowser/Controller.tsx

```diff
@@ -1,36 +1,22 @@
 import React from 'react';
 
-import { useAction } from './context/actions';
 import { useControl } from './context/controls';
 
 /**
  * Defines and provides default `StorageBrowser`
  * loading behavior
  */
 export function Controller(): null {
-  const [{ selected }] = useControl({ type: 'ACTION_SELECT' });
-  const [{ history, path }] = useControl({
+  const [{ history }] = useControl({
     type: 'NAVIGATE',
   });
-  const [, handleListLocationItems] = useAction({
-    type: 'LIST_LOCATION_ITEMS',
-  });
 
   const currentPosition = history.length;
   const hasHistory = !!currentPosition;
 
   React.useEffect(() => {
     if (!hasHistory) return;
-
```

Comment:
Removing this since we're adding a useEffect with handleList() to the LocationDetailView.